doc/parameters/param_list.md

```diff
@@ -479,7 +479,7 @@ When `planner_plugins` parameter is not overridden, the following default plugin
 | ----------| --------| ------------|
 | stop_on_failure | true | Whether to fail action task if a single waypoint fails. If false, will continue to next waypoint. |
 | loop_rate | 20 | Rate to check for results from current navigation task |
-| waypoint_task_executor_plugin | `waypoint_task_executor` | Name of plugin to be loaded for executing waypoint tasks.|
+| waypoint_task_executor_plugin | `WaitAtWaypoint` | Name of plugin to be loaded for executing waypoint tasks.|
```

Comment:
@SteveMacenski, I think it is resolved here ? `waypoint_task_executor`==> `WaitAtWaypoint`